src/main/java/com/assertthat/selenium_shutterbug/core/PageSnapshot.java

```diff
@@ -93,7 +93,7 @@ public PageSnapshot highlightWithText(WebElement element, Color elementColor, in
        try {
            highlight(element, elementColor, 0);
            Coordinates coords = new Coordinates(element, devicePixelRatio);
-           image = ImageProcessor.addText(image, coords.getX(), coords.getY() - textFont.getSize() / 2, text, textColor, textFont);
```

Comment:
Need to keep assignment to image so subsequent chained methods can use updated image.